src/content/docs/en/guides/testing.mdx

```diff
@@ -28,6 +28,19 @@ export default getViteConfig({
 });
 ```
 
+By default, `getViteConfig()` will try to load an Astro config file in your project and apply it to the test environment.
```

Comment:
```suggestion
<p><Since v="4.8.0" /><p>

By default, `getViteConfig()` will try to load an Astro config file in your project and apply it to the test environment.
```

It's not amazing, but we *could* do it this way

Reply:
Ah yeah, that’s why that would be confusing. The “By default…” clause has always been the case — that hasn’t changed. The new part is the second argument allowing you to override that default with inline options. So I guess if anything it would happen mid paragraph?

> By default, `getViteConfig()` will try to load an Astro config file in your project and apply it to the test environment.
> As of Astro 4.8, if you need to customize the Astro configuration applied in your tests, pass a second argument to `getViteConfig()`:

Reply:
Yes, your suggestion works perfectly well!! If you cared, one other option could be to split up into two paragraphs like:


> By default, getViteConfig() will try to load an Astro config file in your project and apply it to the test environment. 

 `<Since v="4.8.0"/>`

> `getViteConfig()` can also receive a second argument to customize the Astro configuration applied in your tests. Pass an object with any additional [configuration options](/en/reference/configuration-reference/):

Reply:
Thanks! Added it to the paragraph — I think that makes more sense than breaking it up.